README.rst

```diff
@@ -79,19 +79,45 @@ and compared the runtimes between CPU and GPU.
 Installation
 ============
 
-This plugin requires Python version 3.6 or above, as well as PennyLane
-and Qulacs. Installation of this plugin, as well as all dependencies, can be done using ``pip``:
+This plugin requires Python version 3.6 or above, as well as PennyLane and
+Qulacs. Installation of this plugin, as well as all dependencies, can be done
+using ``pip``:
 
 .. code-block:: bash
 
-    $ pip install pennylane-qulacs
+    $ pip install pennylane-qulacs["cpu"]
 
-Alternatively, you can install PennyLane-Qulacs from the `source code <https://github.com/PennyLaneAI/pennylane-qulacs>`__
-by navigating to the top directory and running:
+Note that you need to include whether to install the CPU version
+(``pennylane-qulacs["cpu"]``) or the GPU version (``pennylane-qulacs["gpu"]``)
+of Qulacs for it to be installed correctly. Otherwise Qulacs will need to be
+installed separately.
```

Comment:
Not sure what this means?

Reply:
```suggestion
of Qulacs for it to be installed correctly. Otherwise Qulacs will need to be
installed independently:

.. code-block:: bash

    pip install qulacs pennylane-qulacs
```